frontends/web/src/routes/account/summary/initializeall.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, h, RenderableProps } from 'preact';
+import { Amount } from '../../../components/rates/rates';
+import Spinner from '../../../components/spinner/Spinner';
+import { translate, TranslateProps } from '../../../decorators/translate';
+import { apiGet, apiPost } from '../../../utils/request';
+import { apiWebsocket } from '../../../utils/websocket';
+import { AccountInterface } from '../account';
+import { AccountsSummary } from './accountssummary';
+
+interface ProvidedProps {
+    accounts: AccountInterface[];
+}
+
+interface AccountInitialized {
+    [code: string]: boolean;
+}
+
+export interface Totals {
+    [code: string]: Amount;
+}
+
+interface State {
+    initialized: AccountInitialized;
+}
+
+type Props = ProvidedProps & TranslateProps;
+
+class InitializeAllAccounts extends Component<Props, State> {
+    constructor(props) {
+        super(props);
+        const initialized: AccountInitialized = {};
+        this.props.accounts.map((account: AccountInterface) => {
+            initialized[account.code] = false;
+        });
+        this.state = { initialized };
+      }
+
+    private unsubscribe!: () => void;
+
+    public componentDidMount() {
+        this.props.accounts.map((account: AccountInterface) => {
+            this.onStatusChanged(account.code);
+        });
+        this.unsubscribe = apiWebsocket(this.onEvent);
+    }
+
+    private allInitialized() {
+        return Object.keys(this.state.initialized).every(key => this.state.initialized[key]) ? true : false;
+    }
+
+    private onEvent = data => {
```

Comment:
`data` is implicitly `any`, which we would have to fixe before activating `noImplicitAny`.